camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/WorkflowYamlTest.java

```diff
@@ -1315,4 +1314,79 @@ public void testAddPolicyWithDslFromDeployableBlueprint() throws Exception {
         EntityAsserts.assertAttributeEquals(entity, Sensors.newSensor(Object.class, "result"), "yes");
     }
 
+    @Test
```

Comment:
any test (such as this) that actually performs SSH should be marked as `@Test(groups="Integration")` as unit tests shouldn't assume the ability to `ssh localhost`

alternatively it a MockSshLocation can be used (add something like `mock-location` to catalog and pass it)

Reply:
@ahgittin I had this in the back of my mind to do before submitting the PR and I forgot. I annotated the test methods.